lib/MusicBrainz/Server/Form/Filter/Work.pm

```diff
@@ -37,7 +37,8 @@ sub filter_field_names {
 sub options_role_type {
     return [
         { value => 1, label => l('As performer') },
-        { value => 2, label => l('As writer') },
+        { value => 2, label => l('As author') },
+        { value => 3, label => l('As something else') },
```

Comment:
Wouldn't it make sense for these to match the column names if that's what they're filtering by? Right now only "author" matches.